src/NuGet.Clients/PackageManagement.UI/Prompts/DotnetDeprecatedPrompt.cs

```diff
@@ -0,0 +1,71 @@
+﻿// Copyright (c) .NET Foundation. All rights reserved.
+// Licensed under the Apache License, Version 2.0. See License.txt in the project root for license information.
+
+using System;
+using System.Collections.Generic;
+using System.Linq;
+using NuGet.Frameworks;
+using NuGet.ProjectManagement;
+
+namespace NuGet.PackageManagement.UI.Prompts
```

Comment:
To avoid polluting usings like [here](https://github.com/NuGet/NuGet.Client/pull/799/files/4716c3dfdcb88c0207c1e5df03d7a5e01c7e550c#diff-0ff32fcb23b5fceef26b2f2f205bebabR13).
We started enforcing that everywhere else. 
For instance see `NuGet.Protocol.Core.v3`. @emgarten did tremendous job clipping long namespaces there.